app/services/metrics_api/v3/presenters/top_records.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module MetricsApi
+  module V3
+    module Presenters
+      # Presents a +SupplejackApi::TopCollectionMetric+ ready to be returned via the API
+      class TopRecords
+        def initialize(metric)
+          @m = metric
+        end
+
+        def to_json
+          { @m.metric => @m.results }
```

Comment:
do you need to call `#to_json` on the hash?

Reply:
I don't think so, at least that's not how the other presenters implement to_json